src/nvim/testdir/test_startup.vim

```diff
@@ -251,3 +251,16 @@ func Test_silent_ex_mode()
   let out = system(GetVimCommand() . '-u NONE -es -c''set verbose=1|h|exe "%norm\<c-y>\<c-d>"'' -c cq')
   call assert_notmatch('E315:', out)
 endfunc
+
+func Test_default_term()
```

Comment:
```vim
" Nvim 'term' is readonly.
return
```

Let's return early here instead of skipping (AKA `throw "skipped: ..."`), because this behavior is "wontfix". (Probably other "skipped" legacy tests should return early instead of showing up in the "skipped" list, for a similar reason.)

Reply:
Oh, I see you changed the test. I guess that's fine.